__tests__/app.test.js

```diff
@@ -151,3 +151,78 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201 - POST: responds with posted comment when request contains an object of username and body", () => {
+    const commentToPost = {
+      username: "butter_bridge",
+      body: "Live long and prosper",
+    };
+    return request(app)
+      .post("/api/articles/3/comments")
+      .send(commentToPost)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment).toMatchObject([
+          {
+            comment_id: expect.any(Number),
+            body: "Live long and prosper",
+            article_id: 3,
+            author: "butter_bridge",
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          },
+        ]);
+      });
+  });
+  test("404: responds with a message 'Path not found' when article_id path is valid but does not exist", () => {
+    const commentToPost = {
+      username: "butter_bridge",
+      body: "Live long and prosper",
+    };
+    return request(app)
+      .post("/api/articles/49/comments")
+      .send(commentToPost)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Path not found!");
```

Comment:
I might be tempted to have this be just not found rather than path not found, just because in my head "path" means ive probably spelled the end point incorrectly rather than tried to go to a valid path, but with a non-existent review id